src/regex_redux.rs

```diff
@@ -9,32 +9,32 @@
 
 extern crate regex;
 
+use regex::bytes::Regex;
 use std::borrow::Cow;
+use std::error::Error;
 use std::fs;
 use std::sync::Arc;
 use std::thread;
 
-macro_rules! regex { ($re:expr) => { ::regex::bytes::Regex::new($re).unwrap() } }
-
-fn main() {
-    let mut seq = fs::read("/dev/stdin").unwrap();
+fn main() -> Result<(), Box<Error>> {
```

Comment:
I actually do not agree that this is idiomatic. This will print the debug representation of an error instead of the display representation, which is typically not what you want. I do not think exemplar programs on the benchmark game should teach folks to do this.

On the other hand, a bunch of unwraps isn't idiomatic either. If we want to care about doing things idiomatically, then I'd recommend something like this instead:

```rust
fn main() {
    if let Err(err) = try_main() {
        eprintln!("{}", err);
        process::exit(1);
    }
}

fn try_main() -> Result<(), Box<Error>> {
    ...
}
```